lexicons/app/bsky/actor/defs.json

```diff
@@ -343,6 +344,16 @@
             "type": "ref",
             "ref": "app.bsky.actor.defs#mutedWordTarget"
           }
+        },
+        "actorTarget": {
+          "type": "string",
+          "description": "Groups of users to apply the muted word to. If undefined, applies to all users.",
+          "knownValues": ["all", "exclude-following"]
```

Comment:
Ah yeah we do! So interesting thing though: this removes the `?` optionality of the prop `actorTarget`. Technically on first read, mute words aren't migrated, so `actorTarget` for old mute words will always be undefined.

In my latest commit, I opted to map over existing words and insert that default value if `actorTarget` is undefined.

Cool with that? If not, I think the move would be to remove the default to keep the type optional.

Reply:
Huh. I dont recall why we wouldve done it that way, other than perhaps because we expected the default value to always get filled in.

I'm totally good w/what you did but up to you

Reply:
I like that it's a little tighter like this so gonna roll with it 👍